### DISCUSSION TITLE
doris 查询hive外表性能如何？

### DISCUSSION THREAD

**Discussion by NULL-1995** [apache/doris#21595] [A - General / Q&A]
官网介绍是可以作为联邦查询引擎，访问hive数据，但是网上相关资料较少。想确认下查询hive性能如何？doris访问hive外表做了哪些查询优化策略吗？

**Comment by NULL-1995** [comment]
另外hive外表多表关联时的性能呢？

**Comment by yagagagaga** [comment]
Doris 查 Hive 使用的是 Doris 的向量化化引擎和 pipeline 引擎，速度远远超过 Hive 和 Spark。

**Reply by NULL-1995** [reply]
请教下 和presto相比呢？

**Reply by yagagagaga** [reply]
Doris 2.0 比 Trino 372 快
